cumulus/src/designflow/pnr.py

```diff
@@ -85,9 +86,9 @@ def doTask ( self ):
             unicorn.setApplicationName  ( 'cgt')
             unicorn.registerTool        ( Etesian.GraphicEtesianEngine.grab() )
             unicorn.registerTool        ( Katana.GraphicKatanaEngine.grab() )
-            unicorn.registerTool        ( Tramontana.GraphicTramontanaEngine.grab() )
+           #unicorn.registerTool        ( Tramontana.GraphicTramontanaEngine.grab() )
             unicorn.registerTool        ( Bora.GraphicBoraEngine.grab() )
-            unicorn.registerTool        ( Tutorial.GraphicTutorialEngine.grab() )
+           #unicorn.registerTool        ( Tutorial.GraphicTutorialEngine.grab() )
```

Comment:
Is there a reason to leave these lines commented out as opposed to delete entirely?

Reply:
Same as above. It's the uncommented which is temporary.